Directory.Build.targets

```diff
@@ -67,6 +65,12 @@
     <IsPackable Condition=" '$(IsPackageInThisPatch)' != 'true' ">false</IsPackable>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- When OnlyPackPlatformSpecificPackages is set, only produce packages for projects which set RuntimeIdentifier. -->
+    <!-- Keep this below where we set "IsPackageInThisPatch" -->
```

Comment:
If moving this condition does anything, it's not because of `$(IsPackageInThisPatch)`. That property is always `true`.

Reply:
Sorry, "**almost**-always `true`".

But, the comment remains confusing and I didn't get what you were trying to do. Best to make `$(IsPackageInThisPatch)` irrelevant i.e. remove the https://github.com/aspnet/AspNetCore/pull/17969/files#diff-56f8482aeb786dd105fbf93dedf2615fR78-R94 block. We should _never_ use baseline versions except to confirm dependencies remain unchanged patch-to-patch. Every release is now complete or nearly so; the only optional shipping bits are the targeting packs.

Reply:
> We should never use baseline versions

That was my initial thought as well, but I was a little nervous about not 100% understanding the implications. It now looks like we won't reset January for this, so I have more time to get this right.